pkg/scheduler/framework/interface.go

```diff
@@ -48,6 +49,12 @@ type PluginToNodeScores map[string]NodeScoreList
 // NodeToStatusMap declares map from node name to its status.
 type NodeToStatusMap map[string]*Status
 
+// Diagnosis records the details to diagnose a scheduling failure.
+type Diagnosis struct {
```

Comment:
should we have this in types.go with FitError since no framework interface actually return this type.

Reply:
SG.